build.sbt

```diff
@@ -45,10 +45,13 @@ lazy val lib = project
     )
   )
 
+def env(propName: String): String =
+  sys.env.get(propName).filter(_.trim.nonEmpty).getOrElse("DEV")
```

Comment:
`"DEV"` will only be reached when an env var is not found. In reality this will never happen, but we want to return a `String` type here.